inception-project/inception
# #1294 add more feature config switches

**What's in the PR**
- Added switch "recommender.enabled"
- Added switch "recommender.evaluation-page.enabled"
- Added switch "external-search.enabled"
- Added switch "external-search.elastic-search.enabled"
- Added switch "external-search.pub-annotation.enabled"
- Added switch "knowledge-base.enabled"
- Added switch "knowledge-base.fact-linking.enabled"
- Added switch "knowledge-base.entity-linking.enabled"
- Added switch "search.enabled"
- Allow reification only on wikidata-type KBs (remote ones)

**How to test manually**
* Enable/disable the switches in different combinations and see if the application still works as expected

**Automatic testing**

**Documentation**
